src/view/Chart.ts

```diff
@@ -152,7 +152,7 @@ class ChartView {
      * Highlight series or specified data item.
      */
     highlight(seriesModel: SeriesModel, ecModel: GlobalModel, api: ExtensionAPI, payload: Payload): void {
-        toggleHighlight(seriesModel.getData(), payload, 'emphasis');
+        toggleHighlight(seriesModel.getData(payload && payload.dataType), payload, 'emphasis');
```

Comment:
Developers may give wrong dataType that is not supported in the echarts. It's better to check if `data` exists here.

Also `downplay` method should also add this change.